From 16d42ec2e2126b40d4b94368ede8e4f3ed9de594 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Tue, 11 Nov 2014 13:10:51 +1000
Subject: [PATCH] symbols/il: add HOLAM HASER FOR VAV on shift+5 to
 il(biblical)

From the reporter in
https://bugzilla.redhat.com/show_bug.cgi?id=1132511
    The "il biblical" layout does not allow to type the Unicode character
    U+05BA. However, according to the documentation of the layout (Tiro layout
    version 1.5), available on the link provided below, U+05BA should be
    accessible with the combination shift+5 (see page 7 and/or the attached
    screenshot).
    http://www.sbl-site.org/Fonts/BiblicalHebrewTiroManual.pdf

See also: http://codepoints.net/U+05BA

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 symbols/il | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/symbols/il b/symbols/il
index 48452d6..67487c1 100644
--- a/symbols/il
+++ b/symbols/il
@@ -205,7 +205,7 @@ xkb_symbols "biblical" {
 	key <AE02> { [ 2,     U0599, VoidSymbol, at ] };
 	key <AE03> { [ 3,     U0592, VoidSymbol, numbersign ] };
 	key <AE04> { [ 4,     U05AF, NewSheqelSign, dollar ] };
-	key <AE05> { [ 5,     VoidSymbol, U200D, percent ] };
+	key <AE05> { [ 5,     U05BA, U200D, percent ] };
 	key <AE06> { [ 6,     U05B9, U200C, asciicircum ] };
 	key <AE07> { [ 7,     U05BF, U034F, ampersand ] };
 	key <AE08> { [ 8,     U05C2, U200E, asterisk ] };
-- 
2.1.0

